cmd/pebble/main.go

```diff
@@ -29,10 +29,19 @@ type config struct {
 	}
 }
 
+const DefaultConfigPath = "test/config/default-config.json"
+
+func getDefaultConfig() (config) {
```

Comment:
The automation checks are failing because `go fmt` says you don't need parens around the return type.
```suggestion
func getDefaultConfig() config {
```